jenkins-scripts/dsl/ignition_collection.dsl

```diff
@@ -189,7 +188,85 @@ ignition_collection_jobs =
         'sdformat-ci-sdformat9-homebrew-amd64',
         'sdformat-ci-sdformat9-windows7-amd64',
         'sdformat-install-sdformat9_pkg-bionic-amd64'
-  ]
+  ],
+  'dome' : [
+        'ign_gazebo-master-win',
+        'ign_gui-maser-win',
+        'ign_physics-master-win',
+        'ign_rendering-master-win',
+        'ign_sensors-master-win',
```

Comment:
I think all these `ign_` branches use the format `ign_*-ci-win` for the master branch, not `*-master-win`

Reply:
ouch thanks! d56c066

Reply:
one extra change in 7ad2c37 and now it looks good to me